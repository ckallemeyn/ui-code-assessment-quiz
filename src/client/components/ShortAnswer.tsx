import React from 'react'

const ShortAnswer = () => {
  return (
    <>

    </>
  )
}

export default ShortAnswer
