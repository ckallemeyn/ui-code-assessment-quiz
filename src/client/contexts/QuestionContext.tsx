import { createContext } from "react"

// maybe play around with creating an HOC for the entire app
const QuestionContext = createContext({});
QuestionContext.displayName = 'QuestionContext';

export default QuestionContext;
