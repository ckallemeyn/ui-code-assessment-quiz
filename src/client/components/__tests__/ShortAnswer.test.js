import React from "react";
import { render } from "@testing-library/react";
import ShortAnswer from "../ShortAnswer";
import QuestionContext from '../../contexts/QuestionContext'

const renderWithContext = (value) => {
  const updateQuestion = jest.fn();

  return render(
    <QuestionContext.Provider value={value}>
      <ShortAnswer />
    </QuestionContext.Provider>
  )
}

describe("Short Answer", () => {
  test("it should render", () => {
    const four = 4;

    expect(four).toEqual(4);
  });
});

