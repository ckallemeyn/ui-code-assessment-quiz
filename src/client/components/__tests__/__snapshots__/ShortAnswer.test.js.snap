// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component test suite it should render 1`] = `
<div>
  <div
    class="ui input"
  >
    <input
      name="shortAnswer"
      type="text"
      value=""
    />
  </div>
</div>
`;
